__tests__/spelling.js

```diff
@@ -9,10 +9,21 @@ const reporter = require('vfile-reporter')
 
 const describeRule = require('../test-utils/describe-rule')
 
-const ignore = yaml.safeLoad(fs.readFileSync('./__tests__/spelling-ignore.yml', 'utf8'))
+const ignoreWords = yaml.safeLoad(fs.readFileSync('./__tests__/spelling-ignore.yml', 'utf8'))
+// https://www.w3.org/WAI/WCAG21/Techniques
+const ignoreTechniques = [`ARIA`, `C`, `F`, `G`, `H`].reduce((out, techniquePrefix) => {
+	let i = 1
+	while (i < 500) {
+		// Arbitarily chosen number
```

Comment:
```suggestion
		// Arbitrarily chosen number
```
Arbitrarily indeed, but should be harmless so I see no need to fine tune that number now…